src/System.Runtime.TimeZoneData/System.Runtime.TimeZoneData.csproj

```diff
@@ -0,0 +1,31 @@
+<Project Sdk="Microsoft.NET.Sdk">
+    <PropertyGroup>
+      <TimeZoneDataVersion>2020a</TimeZoneDataVersion>
+      <TimeZoneSource>https://data.iana.org/time-zones/tzdb</TimeZoneSource>
+      <InputFolder>$(MSBuildProjectDirectory)/input</InputFolder>
+      <OutputFolder>$(MSBuildProjectDirectory)/zoneinfo</OutputFolder>
+    </PropertyGroup>
+    <ItemGroup>
+        <Area Include="africa" />
+        <Area Include="antarctica" />
+        <Area Include="asia" />
+        <Area Include="australasia" />
+        <Area Include="etcetera" />
+        <Area Include="europe" />
+        <Area Include="northamerica" />
+        <Area Include="southamerica" />
+    </ItemGroup>
+    <Target Name="UpdateData">
+        <DownloadFile
+            SourceUrl="$(TimeZoneSource)-$(TimeZoneDataVersion)/%(Area.Identity)"
+            DestinationFolder="$(InputFolder)">
+        </DownloadFile>
+        <DownloadFile
+            SourceUrl="$(TimeZoneSource)-$(TimeZoneDataVersion)/zone1970.tab"
+            DestinationFolder="$(OutputFolder)">
+        </DownloadFile>
+    </Target>
+    <Target Name="CompileData" BeforeTargets="Build">
+        <Exec Command="zic -d $(OutputFolder) $(InputFolder)/%(Area.Identity)" />
```

Comment:
Presumably this won't work on Windows?

Reply:
No it won't.

Reply:
So that's why CI is failing on Windows. Is there a way to make this work on Windows?

Reply:
No, the `zic` tool is a Unix-specific tool. Can we skip building this project on Windows?

Reply:
an alternative would be to only put the input files into the nuget package and run the `zic` tool as part of the dotnet/runtime WASM build which we know only runs on Unix.